jinahub/indexers/indexer/LMDBIndexer/Dockerfile

```diff
@@ -10,6 +10,6 @@ WORKDIR /workspace
 RUN python3.7 -m pip install -r requirements.txt
 
 # for testing the image
-RUN python3.7 -m pip install pytest && python3.7 -m pytest -s -v tests/test_indexer.py
+RUN python3.7 -m pip install pytest && python3.7 -m pytest -s -v tests/
 
 ENTRYPOINT ["jina", "pod", "--uses", "config.yml"]
```

Comment:
Feel free to ignore and do in a later step: Shouldn't we add the `FROM` `BASE` syntax here as well?

Reply:
Part of later PRs then ;)